app/src/androidTest/java/com/github/se/travelpouch/e2e/ProfileNotificationTest.kt

```diff
@@ -0,0 +1,321 @@
+        composeTestRule.onNodeWithText("Home").assertIsDisplayed()
+
+        // Go to notifications and verify that notification is received + accept
+        composeTestRule.onNodeWithText("Notifications").performClick()
+        composeTestRule
+            .onNodeWithText(
+                "Example6 invited you to join the travel Test Notification Profile as a PARTICIPANT.")
+            .assertIsDisplayed()
+        composeTestRule.onNodeWithText("ACCEPT").assertIsDisplayed().performClick()
+
+        composeTestRule.onNodeWithTag("goBackButton").assertIsDisplayed().performClick()
+
+        // Verify that the travel is displayed and that only two participants
+        composeTestRule.waitUntil(timeoutMillis = 10000) {
+          composeTestRule.onNodeWithText("Test Notification Profile").isDisplayed()
+        }
+        composeTestRule
+            .onNodeWithText("Test Notification Profile")
+            .assertIsDisplayed()
+            .performClick()
+        composeTestRule.onNodeWithTag("settingsButton").assertIsDisplayed().performClick()
+        composeTestRule.onNodeWithTag("plusButton").performClick()
+        composeTestRule.onNodeWithText("Manage participants", useUnmergedTree = true).performClick()
+
+        composeTestRule.onNodeWithText("example5@example.com").assertIsDisplayed()
+        composeTestRule.onNodeWithText("example6@example.com").assertIsDisplayed()
+
+        // Return to the travelList screen to send friend notification
+        composeTestRule.onNodeWithTag("goBackButton").assertIsDisplayed().performClick()
+        composeTestRule.onNodeWithTag("goBackButton").assertIsDisplayed().performClick()
+        composeTestRule.onNodeWithTag("goBackButton").assertIsDisplayed().performClick()
+
+        // Go to profile screen
+        composeTestRule.onNodeWithTag("menuFab").performClick()
+        composeTestRule.onNodeWithText("Profile").performClick()
+
+        // add a friend, which is example6@example.com
+        composeTestRule.onNodeWithTag("settingsButton").assertIsDisplayed().performClick()
+        composeTestRule.onNodeWithText("Add Friend").assertIsDisplayed().performClick()
+        composeTestRule
+            .onNodeWithTag("addingFriendField", useUnmergedTree = true)
+            .assertIsDisplayed()
+            .performTextClearance()
+        composeTestRule
+            .onNodeWithTag("addingFriendField", useUnmergedTree = true)
+            .performTextInput("example6@example.com")
+        composeTestRule.onNodeWithTag("addingFriendButton").assertIsDisplayed().performClick()
+
+        composeTestRule.onNodeWithTag("goBackButton").assertIsDisplayed().performClick()
+        composeTestRule.onNodeWithTag("goBackButton").assertIsDisplayed().performClick()
+
+        composeTestRule.onNodeWithTag("menuFab").performClick()
+        composeTestRule.onNodeWithText("Log out").performClick()
+
+        // After log out, we log in with example6@example.com a accept friend invitation
```

Comment:
```suggestion
        // After log out, we log in with example6@example.com and accept friend invitation
```
Some small typos in the comments, this way it's easier to understand what's going on.